src/libopensc/card-openpgp.c

```diff
@@ -787,6 +788,14 @@ pgp_enumerate_blob(sc_card_t *card, struct blob *blob)
 		}
 		tag |= cla;
 
+		/* Awful hack for composite DOs that have
+		 * a TLV with the DO's id encompassing the
+		 * entire blob. Example: Yubikey Neo */
+		if (tag == blob->id) {
```

Comment:
Scrolling through the standard I can't see that this encoding is valid, right?

If this is Yubikey specific, could you add a check for the card here?